pkg/config/runtime.go

```diff
@@ -87,7 +88,7 @@ func (r *Runtime) resolveRules(bag attribute.Bag, aspectSet AspectSet, rules []*
 	var lerr error
 
 	for _, rule := range rules {
-		if glog.V(2) {
+		if glog.V(3) {
```

Comment:
I think at one point there was another statement inside glog.V(2). Changes to inline form now.